mantisshrimp/backbones/__init__.py

```diff
@@ -1,5 +1,3 @@
 from mantisshrimp.backbones.torchvision_backbones import *
-from torchvision.models.detection.backbone_utils import (
-    BackboneWithFPN,
-    resnet_fpn_backbone,
-)
+from torchvision.models.detection.backbone_utils import *
```

Comment:
Don't import everything from torchvision, we don't control their name space, here we should only import what we need `BackboneWithFPN` and `resnet_fpn_backbone`

Reply:
Changed.